agents/a2a_framework.py

```
@@ -2,260 +2,257 @@
-            'status': 'completed'
+            "negotiation_id": negotiation_id,
+            "participants": agents,
+            "solution": solution,
+            "status": "completed",
         }
-    
-    async def find_optimal_solution(self, proposals: Dict, constraints: Dict) -> Dict:
+
+    async def find_optimal_solution(
+            self,
+            proposals: Dict,
```

Ruff (0.11.9)

Unused method argument: `proposals`

(ARG002)